src/yoroi-wallets/utils/validators.ts

```diff
@@ -199,3 +199,7 @@ export const cleanMnemonic = (mnemonic: string) => {
   // remove leading/trailing whitespace
   return mnemonic.trim()
 }
+
+export const isValidModerationStatus = (status: unknown): status is YoroiNFTModerationStatus => {
+  return Object.values(YoroiNFTModerationStatus).includes(<YoroiNFTModerationStatus>status)
+}
```

Comment:
Used to parse moderation status from rest API

Reply:
:arrow_up:

Reply:
I've now created `parseModerationStatus`